.github/workflows/test.yaml

```
@@ -5,16 +5,24 @@ on:
     name: Unit Test
+    services:
+      redis:
+        image: redis:7-alpine
+        ports:
+          - 6379:6379
+        options: >-
+          --health-cmd "redis-cli -a redis_password ping || redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        cmd: redis-server --requirepass redis_password
```

actionlint (1.7.7)

unexpected key "cmd" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)